test/integration/concrete/lockup/withdraw/withdraw.t.sol

```diff
@@ -144,19 +145,153 @@ abstract contract Withdraw_Integration_Concrete_Test is Integration_Test, Withdr
         whenToNonZeroAddress
         whenWithdrawAmountNotZero
         whenNoOverdraw
+        whenCallerRecipient
     {
+        test_Withdraw_CallerRecipient(defaultStreamId, users.sender);
+    }
+
+    modifier givenSenderContract() {
+        _;
+    }
+
+    function test_Withdraw_SenderContract()
```

Comment:
this should be `test_Withdraw_SenderDoesNotImplementHook`, since sender is `noop` which does not implement hook. And there has already been a modifier `givenSenderContract` used.

Reply:
correct, missed it